solr/core/src/java/org/apache/solr/cloud/autoscaling/InactiveShardPlanAction.java

```diff
@@ -102,9 +104,14 @@ public void process(TriggerEvent event, ActionContext context) throws Exception
         String parentPath = ZkStateReader.COLLECTIONS_ZKNODE + "/" + coll.getName();
         List<String> locks;
         try {
-          locks = cloudManager.getDistribStateManager().listData(parentPath).stream()
-              .filter(name -> name.endsWith("-splitting"))
-              .collect(Collectors.toList());
+          DistribStateManager stateManager = cloudManager.getDistribStateManager();
```

Comment:
this change is an unrelated fix, please ignore for now - this should go into a separate Jira.